biome.json

```diff
@@ -385,6 +385,18 @@
           }
         }
       }
+    },
+    {
+      "assist": { "actions": { "source": { "organizeImports": "on" } } },
+      "includes": ["**/*.js"]
```

Comment:
Should we not have this turned on for `*.js` files? Or we can remove https://github.com/Comcast/react-data-grid/blob/main/.prettierignore#L1-L3 and let prettier handle everything. Wdyt?